### ISSUE TITLE
January 2024 Dependency Update

### ISSUE THREAD

**Issue by mike-weiner** [mike-weiner/wlbot#10]
The project's dependencies are severely out of date. They need to be updated to get a new release published. Once that is complete, let's look into dependabot scanning so we can automate keeping dependencies update-to-date and doing a monthly release.

**Comment by mike-weiner** [comment]
Closing as deps. updates have been merged into `development` and will be in next release.
